samples/ChatApp/ChatApp.Server/ChatHub.cs

```diff
@@ -33,20 +33,35 @@ public async Task LeaveAsync()
             this.Broadcast(this.room).OnLeave(this.myName);
         }
 
-
         public async Task SendMessageAsync(string message)
         {
-            var response = new MessageResponse {  UserName = this.myName, Message = message };
+            var response = new MessageResponse { UserName = this.myName, Message = message };
             this.Broadcast(this.room).OnSendMessage(response);
 
             await Task.CompletedTask;
         }
 
+        public Task GenerateException(string message)
+        {
+            throw new Exception(message);
+        }
+
+        public Task UnaryGenerateException(string message)
```

Comment:
Please delete it because it is not used.

Reply:
done.